### PR TITLE
Changes header texts

### PR DESCRIPTION
**Pull request by janjaap**
This PR changes signaling column header texts.

### CODE REVIEW

**Review by MauritsR — CHANGES_REQUESTED** [review]
If you remove the end date that is used in the request for this data, the data will reflect the updated text.

**Comment by janjaap** [comment]
> If you remove the end date that is used in the request for this data, the data will reflect the updated text.

Done

File: src/signals/incident-management/containers/Signaling/Signaling.tsx [hunk 1]
```diff
@@ -49,7 +47,7 @@ const Signaling: FunctionComponent = () => {
   } = useGetReportReopenRequested()
 
   useEffect(() => {
-    getReportOpen({ end: endOpen })
+    getReportOpen({ end: new Date().toISOString() })
```
Comment (MauritsR) [inline src/signals/incident-management/containers/Signaling/Signaling.tsx:50]: The end parameter is optional in the API, so the addition is unnecessary. 

Looks like `getReportOpen` should be typed differently and/or set a default parameter to handle empty input.

**Review by MauritsR — DISMISSED** [review]

File: src/signals/incident-management/containers/Signaling/Signaling.tsx (hunk 1 above)
Comment (janjaap) [reply src/signals/incident-management/containers/Signaling/Signaling.tsx]: Good catch. Changed it.

**Pull request merged**